modtools/item-trigger.lua

```diff
@@ -48,14 +49,15 @@ Arguments::
         trigger the commmand on items with the given material
         examples
             INORGANIC:IRON
-            CREATURE_MAT:DWARF:BRAIN
-            PLANT_MAT:MUSHROOM_HELMET_PLUMP:DRINK
+            CREATURE:DWARF:BRAIN
```

Comment:
Not familiar with this script... does this break backwards compatibility?

Reply:
I suppose so? The issue is that dfhack.matinfo.decode(item):getToken() returns strings in the format displayed above, which is incongruous with the format previously documented in this script's -help printout. Specifying something like "-material CREATURE_MAT:DWARF:BRAIN" shouldn't have worked at all in recent versions (tested it as far back as 0.43.05 to check), and the fact that this has gone unreported for so long indicates that the script probably hasn't seen much use in this regard.

Reply:
Oh, I missed that part in the PR description, and thought you had actually changed the format here. I really don't know much about the modtools scripts, so thanks for putting in the time to work on them.